docs/CODE_STYLE.md

```
@@ -45,18 +46,15 @@ All variable names, function names, etc., should use `snake_case` unless stated
 **Macros**:
-**Macros**:
-
 - Written in all caps `SNAKE_CASE`.
 - Try to denote their group using a short prefix (e.g., `PREFIX_`).
 - Example: `#macro COL_DARK_RED`.
 
 **Enums**:
- **Enums**:
-
- - Enum names should start with an `e` prefix and be in all caps `SNAKE_CASE`.
+ - Enum names should start with an `e` prefix.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 0; Actual: 1

(MD007, ul-indent)